apim/3.x/templates/api/api-configmap.yaml

```diff
@@ -75,6 +75,12 @@ data:
         {{- end }}
       {{- end }}
 
+    console:
+      api:
+        url: "https://{{index .Values.api.ingress.management.hosts 0 }}{{ .Values.api.ingress.management.path }}"
+      ui:
+        url: "https://{{index .Values.ui.ingress.hosts 0 }}{{ .Values.ui.ingress.path }}"
```

Comment:
this results in 

```yaml
    console:
      ui:
        url: "https://api-portal.gravitee.local/console(/|$)(.*)"
```

for my local test setup, chis is not a valid URL.

@phiz71 iny suggestion?

Reply:
resolved issue via regex

Reply:
Ok, I checked with @brasseld.
Could you please:
 - remove `(/|$)(.*)` from the values.yml file.
 ```
 path: /console(/|$)(.*) ==>  path: /console
 ```
 - add `(/|$)(.*)` in ui-ingress.yaml file
 ```
path: {{ $ingressPath }} ==>  path: {{ $ingressPath }}(/|$)(.*)
 ```
 ?

Then you will be able to use `{{ .Values.ui.ingress.path }}` without regex filter